docs/en/api/POSIX-API.md

```diff
@@ -607,3 +607,46 @@ $ cd async-profiler && ./profiler.sh -e lock -d 30 -f lock.txt `jps | grep Allux
 - `-d` define the duration. Try to cover the whole POSIX API testing duration
 - `-e` define the profiling target
 - `-f` define the file name to dump the profile information to
+
+## Fuse Shell Tool
+
+The Alluxio JNI-Fuse client provides a useful shell tool to perform some internal operations, such as clearing the client metadata cache. 
+If our Alluxio-Fuse mount point is `/mnt/alluxio-fuse`, the command patten of Fuse Shell is:
+```console
+$ ls -l /mnt/alluxio-fuse/.alluxiocli.[COMMAND].[SUBCOMMAND]
+```
+Among them, the `/.alluxiocli` is the identification string of Fuse Shell, `COMMAND` is the main command (such as `metadatacache`), and `SUBCOMMAND` is the subcommand (such as `drop, size, dropAll`).
+Currently, Fuse Shell only supports `metadatacache` command to clear cache or get cache size, and we will expand more commands and interactive methods in the future.
+To use the Fuse shell tool, alluxio.fuse.special.command.enabled needs to be set to true in ${ALLUXIO_HOME}/conf/alluxio-site.properties before launching the Fuse applications:
+```console
+$ alluxio.fuse.special.command.enabled=true
+```
+
+### Metadatacache Command
+
+When we use Alluxio fuse, we usually turn on the client metadata cache. For example, in a scenario that reads a large number of small files such as AI, enabling client metadata caching can relieve Alluxio Master's metadata pressure and improve read performance.
```

Comment:
When we use Alluxio fuse, we usually turn on the client metadata cache. 
-> 
Client-side metadata cache can be enabled by setting `....=true` to reduce the latency of metadata cache operations and improve FUSE performance in many workloads.

Reply:
Done!